lib/common/common.go

```diff
@@ -51,14 +51,10 @@ func ParseDockerURL(arg string) *types.ParsedDockerURL {
 	}
 	indexURL, imageName := SplitReposName(taglessRemote)
 
-	if indexURL == "" && !strings.Contains(imageName, "/") {
+	if indexURL == defaultIndexURL && !strings.Contains(imageName, "/") {
```

Comment:
Is this something specific to the registry at docker.io, but not to the one at, say, localhost?